### PR TITLE
sign-in page design improvements

### PR DESCRIPTION
**Pull request by valeriegc**
Improved the overall design on the sign in page. In future edits the sign-in functionality needs to still be added - used pure CSS to make icons.

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,72 +1,114 @@
...
+<script>
+	import Lock from '../icons/Lock.svelte';
+	import UserIconSmall from '../icons/UserIconSmall.svelte';
+</script>
+
+<main>
+	<div class="upperHalf" />
 	<header>
-		<h1>Image-map Generator</h1>
+		<h1>I M A G E &nbsp M A P</h1>
```
Comment (AlexanderKolberg) [inline src/routes/+page.svelte]: letter-spacing: .2rem; ?

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -1,72 +1,114 @@
...
+		border: none;
+		background-color: rgba(255, 255, 255, 0.7);
+		font-size: 1.5rem;
+		padding: 10px;
+	}
+	input::placeholder {
 		color: #000099;
-		height: 100px;
+		text-align: center;
+	}
```
Comment (AlexanderKolberg) [inline src/routes/+page.svelte:87]: Should this be global?

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -1,72 +1,114 @@
...
+	.userIconContainer {
+		position: absolute;
+		left: -8px;
+		top: -8px;
+	}
+	.loginButton {
+		height: 60px;
+		width: 295px;
+		background-color: whitesmoke;
```
Comment (AlexanderKolberg) [inline src/routes/+page.svelte]: Is this in your color pallet?

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -1,72 +1,114 @@
...
+		color: #000099;
```
Comment (AlexanderKolberg) [inline src/routes/+page.svelte]: Why are you not using the variable here?

**Review by AlexanderKolberg — CHANGES_REQUESTED** [review]

File: src/routes/+page.svelte (hunk 1 above)
Comment (valeriegc) [reply src/routes/+page.svelte]: thank you, resolved.

File: src/routes/+page.svelte (hunk 2 above)
Comment (valeriegc) [reply src/routes/+page.svelte:87]: nope. there is a diferent place holder elsewhere as well as a different input field

File: src/routes/+page.svelte (hunk 3 above)
Comment (valeriegc) [reply src/routes/+page.svelte]: fixed.

File: src/routes/+page.svelte (hunk 4 above)
Comment (valeriegc) [reply src/routes/+page.svelte]: fixed

**Review by AlexanderKolberg — APPROVED** [review]
LGTM

**Pull request merged**